ui/scss/component/_form-field.scss

```diff
@@ -120,3 +114,24 @@ fieldset-group {
 .form-field--address {
   min-width: 18em;
 }
+
+.form-field__help {
+  @extend .help;
+  margin-top: var(--spacing-miniscule);
+  margin-bottom: var(--spacing-small);
+}
+
+.form-field__conjuction {
+  padding-top: 1rem;
+}
+
+fieldset-section {
+  .form-field__internal-option {
+    margin-top: var(--spacing-small);
+    margin-left: 2.2rem;
+
+    &:first-of-type {
+      margin-top: var(--spacing-small); // Extra specificity needed here since _section.scss is applied after this file
```

Comment:
not blocking to fix this, but this can presumably be eliminated by refactoring so that these rules do not overlap